src/Modules/quorum.ts

```diff
@@ -30,19 +33,30 @@ type Action =
   | { type: "SELECT_FAILURE"; data: HaltingFailure }
   | { type: "UNKNOWN" };
 
-export function fetchQuorum(): Action {
-  return showExample("preHalt");
+export function fetchQuorum() {
+  return async (dispatch: Dispatch) => {
+    const response = await axios.get("http://localhost:8080/quorum");
```

Comment:
Do we use any sort of pre-rendering? We'll likely not always have `localhost` as the server's URL, so it'd be awesome if we could make this configurable somehow.